cachi2/core/package_managers/pip.py

```diff
@@ -1496,8 +1487,8 @@ def _download_pypi_package(requirement, pip_deps_dir, pypi_url, pypi_auth=None):
     Does not download any dependencies (implied: ignores extras). Ignores environment
     markers (target environment is not known to Cachi2).
 
-    :param PipRequirement requirement: PyPI requirement from a requirement.txt file
-    :param Path pip_deps_dir: The deps/pip directory in a Cachi2 request bundle
+    :param requirement: PyPI requirement from a requirement.txt file
```

Comment:
Should the type annotation be removed from here (since we're not adding it to the function signature)?

Reply:
Good catch, fixed here and in PipRequirementsFile